lightning/src/onion_message/messenger.rs

```diff
@@ -1053,20 +1100,49 @@ where
 		)
 	}
 
-	fn handle_onion_message_response<T: OnionMessageContents>(
+	/// Handles the response to an [`OnionMessage`] based on its [`ResponseInstruction`],
+	/// enqueueing any response for sending.
+	///
+	/// This function is useful for asynchronous handling of [`OnionMessage`]s.
+	///
+	/// Handlers have the option to return [`ResponseInstruction::NoResponse`], indicating that
+	/// no immediate response should be sent. Instead, they can transfer the associated [`Responder`]
```

Comment:
"Instead" indicates an alternative to what was just stated. So use "Then" instead (no pun intended).

Reply:
I think that makes more sense.
Used "then" instead of "instead" in [d8adbb0](https://github.com/lightningdevkit/rust-lightning/commit/d8adbb0a5b7dcc39a41596274129ff3d69ec08de) (pun intended :)